extensions/wikia/Recirculation/services/ParselyDataService.class.php

```diff
@@ -32,11 +33,11 @@ public function getPosts( $type, $count = self::PARSELY_POSTS_LIMIT ) {
 			return [];
 		}
 
-		$memcKey = wfSharedMemcKey( __METHOD__, $type, $metadata['key'], $count, self::MCACHE_VER );
+		$memcKey = wfSharedMemcKey( __METHOD__, $type, $metadata['key'], $count, static::MCACHE_VER );
 
 		$data = WikiaDataAccess::cache(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$data". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)